spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -65,19 +65,35 @@ $ git clone https://github.com/{github-repo}
 $ cd spring-cloud-dataflow-server-kubernetes
 $ git checkout {github-tag}
 ----
+
+==== Choosing a Transport
+
+For deployed applications to communicate with each other, a transport middleware needs to be selected. The sample deployment and service YAML files provide configurations for RabbitMQ and Kafka. Only one transport needs to be configured.
```

Comment:
"a transport middleware needs to be selected" -> "a message broker needs to be selected"

Reply:
Possibly similar change to: "Only one transport needs to be configured"